lib/perl/Genome/Model/Tools/Bwa/RunMem.pm

```diff
@@ -403,7 +403,10 @@ sub _sort_commands {
     my $comp_level = 0;
     my $threads = $self->num_threads;
     my $per_thread_mem = $self->_sort_memory_per_thread_mb;
-    my $tmp_path = File::Spec->catfile($self->temp_dir, "samtools-sort");
+    my $tmp_path = $self->temp_dir;
+
+    $tmp_path = Genome::Sys->create_temp_file_path unless $tmp_path;
```

Comment:
I was wondering that as well. I think this code is still necessary based on the documentation though.